thriftserver/server/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <artifactId>livy-main</artifactId>
     <groupId>org.apache.livy</groupId>
-    <version>0.7.1.3.2.3.0-SNAPSHOT</version>
+    <version>0.7.1.3.2.2.0-1095</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
```

Comment:
is missing a dependency - Was this https://github.com/acceldata-io/livy/commit/ee434b1d2897deb92fae9201861a8a7ef108844e applied cleanly? 
```
<dependency>
      <groupId>org.apache.commons</groupId>
      <artifactId>commons-lang3</artifactId>
      <version>3.14.0</version>
    </dependency>
```